.konflux/operator/konflux.Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG OPERATOR_VERSION
```

Comment:
same here with regards to file name prefix

Reply:
see https://github.com/openshift-kni/numaresources-operator/pull/1272#discussion_r2077774193